spec/customer_order_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'customer_order'
+
+describe CustomerOrder  do
+  
+  subject(:customer_order) { described_class.new }
+
+  context 'initialiazation' do
+    it 'defaults to an empty order' do
+      expect(customer_order.order_dishes).to eq Hash.new(0)
+    end
+
+    it 'defaults to a total of 0.00' do
+    end
+  end
+
+  context '#add_items' do
+    it { is_expected.to respond_to :add_items }
+
+    # it 'should add the value to the total' do
+    # end
+
+
+    it 'should add lines to the order' do
+      expect{ customer_order.add_items(:dish_one, 10) }.to change { customer_order.order_dishes.length }.by 1
```

Comment:
Metrics/LineLength: Line is too long. [109/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)